### PR TITLE
Fixes #26076, you can't cheat with eyewear/eye-implants

### LINKED ISSUE #26076: Thermal goggles work with advanced camera consoles

### PR DESCRIPTION
**Pull request by Davidj361**
Fixes #26076
:cl: Davidj361
fix: You can't cheat with eyewear/eye-implants when using advanced cameras
/:cl:

[why]: See https://github.com/tgstation/tgstation/issues/26076
Thanks to MrStonedOne for helping.

### CODE REVIEW

File: code/game/machinery/computer/camera_advanced.dm [hunk 1]
```diff
@@ -92,6 +92,13 @@ obj/machinery/computer/camera_advanced/attack_ai(mob/user)
 	var/eye_initialized = 0
 	var/visible_icon = 0
 	var/image/user_image = null
+	
+/mob/camera/aiEye/remote/update_remote_sight(mob/living/user)
+	user.see_invisible = SEE_INVISIBLE_LIVING //can't see ghosts through cameras
+	// NOTE: I took out a case for isXray() since we don't have that proc in here
```
Comment (optimumtact) [inline code/game/machinery/computer/camera_advanced.dm]: this comment is not required, remove it

**Comment by KorPhaeron** [comment]
what if I take my glasses off/put them back on while using the console?

**Comment by Davidj361** [comment]
@KorPhaeron It doesn't break.

**Comment by Davidj361** [comment]
Christ

**Pull request merged**